### PR TITLE
Update 4.6.0 release notes

### PR DESCRIPTION
**Pull request by xmcclure**

### CODE REVIEW

File: docs/about-mono/releases/4.6.0.md [hunk 1]
```diff
@@ -114,8 +114,30 @@ By doing so Mono runtime will not be able to output IL offsets instead of native
 
 `mono-symbolicate` can be found [here](https://github.com/mono/mono/tree/mono-4.6.0-branch/mcs/tools/mono-symbolicate)
 
+### New default GC bridge processor on Android
+
+The Android implementation makes use of a "bridge processor" which coordinates communication between the Mono and Java garbage collectors. There are three bridge processor implementations included, "old", "new" and "tarjan". In 4.6.0 the default has been changed from "new" to "tarjan". This should result in performance improvements during GC for apps that had not already selected the "tarjan" collector.
```
Comment (akoeplinger) [inline docs/about-mono/releases/4.6.0.md]: What does this mean for Desktop Mono? Can I switch to tarjan as well? Should I? If not, why not? 😄

File: docs/about-mono/releases/4.6.0.md [hunk 2]
```diff
@@ -114,8 +114,30 @@ By doing so Mono runtime will not be able to output IL offsets instead of native
...
+
+### FullAOT: Generic Value Type Sharing on x86-64
+
+The gsharedvt capability, which allows certain complex uses of generics to be used in FullAOT mode, is now available on the x86-64 ISA on Mac OS X and Linux-based platforms. This means gsharedvt is now available on all platforms where FullAOT itself is supported.
+
+### New `disable_omit_fp` option for compiler
```
Comment (akoeplinger) [inline docs/about-mono/releases/4.6.0.md]: nit: add JIT/AOT before compiler to make it clear you're not talking about mcs

File: docs/about-mono/releases/4.6.0.md (hunk 1 above)
Comment (xmcclure) [reply docs/about-mono/releases/4.6.0.md]: Well, it has an impact if you're using java.interop, but that hasn't been released yet so I don't know what to say about that here…

Comment (akoeplinger) [reply docs/about-mono/releases/4.6.0.md]: java.interop is already open source: https://github.com/xamarin/java.interop ?

**Pull request merged**